quill-cassandra-ce/src/main/scala-2.12/io/getquill/CassandraCeContext.scala

```diff
@@ -0,0 +1,101 @@
+package io.getquill
+
+import com.datastax.driver.core._
+import io.getquill.context.cassandra.CqlIdiom
+import io.getquill.util.{ ContextLogger, LoadConfig }
+import cats.effect._
+import cats._
+import io.getquill.util.GuavaCeUtils._
+import cats.implicits._
+import com.typesafe.config.Config
+import io.getquill.context.ce.CeContext
+import fs2.Stream
+
+import scala.jdk.CollectionConverters._
+import scala.language.higherKinds
+
+class CassandraCeContext[N <: NamingStrategy, F[_]: FlatMap](
```

Comment:
I believe this `FlatMap` is redundant with the `Async`.

Reply:
it is

Reply:
Should it be `F[_]: Async`?

Reply:
So this is supposed to be `F[_]: Async`?

Reply:
Async dependcy is this case is passed as `implicit val`. FlatMap is redundant.

Reply:
Okay, cool. Let's drop it then.